android/src/main/java/com/twilio/twilio_voice/BackgroundCallJavaActivity.java

```diff
@@ -198,6 +216,7 @@ private void callCanceled() {
 
     @Override
     protected void onDestroy() {
+        deactivateSensor();
```

Comment:
thought: what deactivateSensor() does is similar to the code below, shouldn't this only be called once?